### ISSUE TITLE
Reduce the amount of jobs run on lampsrv26.epfl.ch

### ISSUE THREAD

**Issue by smarter** [lampepfl/dotty#6641]
I've observed lampsrv26 running out of memory several times now: http://tresormon.epfl.ch/munin/epfl.ch/lampsrv26.epfl.ch/memory.html
lampsrv26 has 128 GB of ram like lampsrv9, but it has 40% more CPU cores, and vulpix will run as many tests in parallel as we have cores, so we naturally end up using more memory. Reducing the number of jobs that drone runs on the machine should help (though it still seems surprising that we end up using that much memory).

@Fnux If you have time, maybe something you could look at ?

**Comment by anatoliykmetyuk** [comment]
Also IMO it would be nice to have a definite way to kill a build immediately. E.g. [this](http://dotty-ci.epfl.ch/lampepfl/dotty/13455) build took one hour to run the `test_sbt` task. I don't remember what it showed when I first cancelled it, but it continued to run, and my subsequent attempts to cancel it resulted in "Failed to cancel your build".

**Comment by nicolasstucki** [comment]
@anatoliykmetyuk, is this still an issue?

**Comment by anatoliykmetyuk** [comment]
Nope, drone is running for releases only these days and GH actions runs 5 jobs per machine. Didn't notice any issues so far with performance.
